docs/appguide.rst

```diff
@@ -68,16 +70,16 @@ any change in its lifecycle.
         BACKGROUND -> START [ label=" __del__()   " ];
         BACKGROUND -> ACTIVE [ label=" foreground()   " ];
         ACTIVE -> BACKGROUND [ label=" background()   " ];
-        ACTIVE -> GO_TO_CLOCK [ label=" sleep() -> False   " ];
-        GO_TO_CLOCK -> BACKGROUND [ label=" background()   " ];
-        ACTIVE -> SLEEPING [ label=" sleep() -> True   " ];
-        SLEEPING -> ACTIVE [ label=" wake()   " ];
+        ACTIVE -> RET_TO_CLOCK [ label=" sleep() -> False   " ];
+        RET_TO_CLOCK -> BACKGROUND [ label=" background()   " ];
+        ACTIVE -> SYSTEM_SLEEPING [ label=" sleep() -> True   " ];
+        SYSTEM_SLEEPING -> ACTIVE [ label=" wake()   " ];
 
         START [ shape=box ];
         BACKGROUND [ shape=box, style=rounded ]
         ACTIVE [ shape=box, style=rounded ]
-        SLEEPING [ shape=box, style=rounded ]
-        GO_TO_CLOCK [ label="GOTO ClockApp" ];
+        SYSTEM_SLEEPING [ shape=box, style=rounded ]
+        RET_TO_CLOCK [ label="Return to ClockApp" ];
```

Comment:
I'm afraid I'm also not a fan of this rename either. States are UPPERCASE in this diagram (to distinguish them from the edges which are app API entry points). Also "Return" implies some kind of backstack (like Android) that wasp-os doesn't have.

Reply:
Alright, I'll remove it